pkg/cmd/clustertask/describe.go

```diff
@@ -262,11 +270,7 @@ func sortResourcesByTypeAndName(tres []v1beta1.TaskResource) []v1beta1.TaskResou
 }
 
 func askClusterTaskName(opts *options.DescribeOptions, p cli.Params) error {
-	cs, err := p.Clients()
-	if err != nil {
-		return err
-	}
-	clusterTaskNames, err := allClusterTaskNames(cs)
+	clusterTaskNames, err := clustertask.GetAllClusterTaskNames(p)
```

Comment:
A call may have already been made to `GetAllClusterTaskNames(p)` before this. Maybe we could pass the list of names to `askClusterTaskName` as an arg by adding a `[]string` param? 

In `askClusterTaskName`, you could check if the list is empty to determine whether to make the call here to avoid doing it twice in some cases.